documentation/modules/ROOT/pages/integrations/tracing.adoc

```diff
@@ -96,6 +96,12 @@ When an outbox even is emitted the extension will:
 * inject the span context into that `java.util.Properties` instance that is serialized into the `tracingspancontext` column
 * write the record into the database
 
+[NOTE]
+====
+The `smallrye-opentracing` dependency is an optional dependency in the Outbox extension. It can be configured by setting an option `quarkus.debezium-outbox.tracing.enabled=true`
+in the Quarkus `application.properties` file.
```

Comment:
I'm not sure whether I get this sentence right, but by default, the sole presence of the OpenTracing extension in the build should enable the tracing usage in the Outbox extension. I.e. no explicit enabled=true setting should be required for that. Only to *disable* tracing despite the OpenTracing extension being present should require an explicit setting. You should find precedence for this in the Quarkus Kafka Streams extension (which does the same for the health extension); alternatively, @Naros or the folks from Quarkus community should be able to help.

Reply:
So @ani-sha I think all we need to change is the default value for the `quarkus.debezium-outbox.tracing.enabled` config option should be _true_ and then we should adjust the documentation based on Gunnar's comments.  This should align with what Gunnar is asking I believe.